dm-setup/student-vm-startup.sh

```diff
@@ -10,6 +10,11 @@ DEPLOYMENT_NAME=`metadata_value "instance/attributes/deployment"`
 apt-get update
 apt-get install -y git kubectl
 
+# Install Docker
```

Comment:
I think we need to move this to later in the script or check for the completion of the account agent. Otherwise there is a chance that the user account daemon has run and the users account doesn't exist yet.

Reply:
In my testing, I didn't have the correct perms to use the Docker socket.

Reply:
It looks like the error is that usermod is not installed in the VM.

```shell
viglesias@student-vm-adv-bc-1529358984:~$ for user in $(grep '^google-sudoers:' /etc/group | cut -f 4 -d : | awk 'BEGIN { RS= "," } {print}'); do usermod -a -G docker $user;done
-bash: usermod: command not found
-bash: usermod: command not found
-bash: usermod: command not found
-bash: usermod: command not found
viglesias@student-vm-adv-bc-1529358984:~$
```

Reply:
Scratch that its installed but (rightly) only `root` can use it. After running the command again it worked.